#--------------------------------------------------------------------------------------------
# IMU Parameters
#--------------------------------------------------------------------------------------------
ScaleX: 1.01632653
OffsetX: -0.242875
ScaleY: 1.018239796
OffsetY: -0.165925
ScaleZ: 1.010637755
OffsetZ: 0.04475
